l1-contracts/bootstrap.sh

```diff
@@ -53,7 +54,6 @@ function build {
 function test_cmds {
   echo "$hash cd l1-contracts && solhint --config ./.solhint.json \"src/**/*.sol\""
   echo "$hash cd l1-contracts && forge fmt --check"
-  echo "$hash cd l1-contracts && forge test --no-match-contract UniswapPortalTest && ./bootstrap.sh gas_report"
```

Comment:
this has disabled tests?

Reply:
Damn, good catch. I just restored the test line, removing the gas_report.